### ISSUE TITLE
tsc unexistant-file -> no error

### ISSUE THREAD

**Issue by trufae** [Microsoft/TypeScript#30544]
**TypeScript Version:** Version 1.5.3

**Expected behavior:**

```
$ tsc jkalsfjklasdf
Cannot open jkalsfjklasdf
$ echo $?
1
```

**Actual behavior:**

```
$ tsc jkalsfjklasdf
$ echo $?
0
```

**Comment by MartinJohns** [comment]
You might want to update your TypeScript version. The latest version (and likely many versions before) report an error.

And I doubt the TypeScript team is going to fix this for the version 1.5.3 which was released **4 years ago**.

**Comment by trufae** [comment]
Omg it was a clean installation on a new machine. Wonder why i had this version.. yeah seems fixed now
